ECUtbildning-WIN20/Patients
# Features/register patient mackans95

# Description

Added an .sql file to recreate database, implemented code to add a patient to the database, and added new class Patient under Domain folder.

## Verification

1. Clone to local repository and run SQL commands in Patients.sql file.
2. Make sure proper nugets are installed for SQL usage.
3. Run application, choose Add patient option, and enter information accordingly.
4. Verify that the patient is now successfully stored in database.
